app/controllers/admin/enterprises_controller.rb

```diff
@@ -375,6 +378,19 @@ def load_properties
       @properties = Spree::Property.pluck(:name)
     end
 
+    def load_tag_rule_types
+      # Load rule types
+      @tag_rule_types = [
+        { id: "FilterShippingMethods", name: t('js.tag_rules.show_hide_shipping') },
+        { id: "FilterPaymentMethods", name: t('js.tag_rules.show_hide_payment') },
+        { id: "FilterOrderCycles", name: t('js.tag_rules.show_hide_order_cycles') }
```

Comment:
Now these translations might not need to be under the `js` namespace. 

Question, do we want these tag rule types to be defined in ruby in the long run, or would we move them back to JS later? 
I'm not really sure, but given that they are directly related to the angular implementation, I think they could move back to JS later. Did you have anything else in mind?